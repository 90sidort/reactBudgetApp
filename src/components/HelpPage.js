import React from 'react'

const HelpPage = () => (
    <div className="content-container">
        <p>This is a simple demo app to show skills in basic React, Redux, unit testing (Jest and Enzyme) and functional testing (comming soon).</p>
        <p>It is meant to keep track of user expenses that can be added via Add expense, read in Dashboard sectiona, edited, filtered, sorted and removed.</p>
    </div>
)

export default HelpPage