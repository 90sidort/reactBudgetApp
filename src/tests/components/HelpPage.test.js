import { shallow } from 'enzyme'
import React from 'react'
import HelpPage from '../../components/HelpPage'

test('Should render HelpPage correctly', () => {
    const wrapper = shallow(<HelpPage/>)
    expect(wrapper).toMatchSnapshot()
})