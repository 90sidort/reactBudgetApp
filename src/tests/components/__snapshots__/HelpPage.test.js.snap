// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render HelpPage correctly 1`] = `
<div
  className="content-container"
>
  <p>
    This is a simple demo app to show skills in basic React, Redux, unit testing (Jest and Enzyme) and functional testing (comming soon).
  </p>
  <p>
    It is meant to keep track of user expenses that can be added via Add expense, read in Dashboard sectiona, edited, filtered, sorted and removed.
  </p>
</div>
`;
